crates/aptos-rosetta/src/types/objects.rs

```diff
@@ -546,21 +542,28 @@ fn parse_operations_from_txn_payload(
 ) -> Vec<Operation> {
     let mut operations = vec![];
     if let TransactionPayload::EntryFunctionPayload(inner) = payload {
-        if AccountAddress::ONE == *inner.function.module.address.inner()
-            && coin_module_identifier() == inner.function.module.name.0
-            && transfer_function_identifier() == inner.function.name.0
-        {
-            if let Some(MoveType::Struct(MoveStructTag {
-                address,
-                module,
-                name,
-                ..
-            })) = inner.type_arguments.first()
-            {
-                if *address.inner() == AccountAddress::ONE
-                    && module.0 == aptos_coin_module_identifier()
-                    && name.0 == aptos_coin_resource_identifier()
+        match (
+            *inner.function.module.address.inner(),
+            inner.function.module.name.0.as_str(),
+            inner.function.name.0.as_str(),
+        ) {
+            (AccountAddress::ONE, COIN_MODULE, TRANSFER_FUNCTION) => {
```

Comment:
What about account::transfer?

Reply:
Interesting... I guess I missed it earlier.  Will fix now

Reply:
This only occurs on failed transactions, so it actually doesn't matter if it shows (it's just for helpful information later)